perception/sub8_pointcloud/include/OGridRegionType.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
I can help you out with including headers from different packages, lol don't be intimidated by cmake.

We should talk about this design more in person. I see your point, but it seems like these enums will be encoding the same information.